news/2021-11-08-performance-points-star-rating-updates.md

```diff
@@ -0,0 +1,132 @@
+
+### Speed hard cap removal
+
+The speed hard cap in osu! has been removed by [**emu1337**](https://osu.ppy.sh/users/2185987) and [**Apo11o**](https://osu.ppy.sh/users/9558549) in [this pull request](https://github.com/ppy/osu/pull/14617), allowing 300+ BPM streams to be weighed accurately. 
+
+The speed cap was previously set to 300 BPM to prevent abuse cases, which means that calculating higher BPMs requires consideration into those cases. Two factors are implemented with regard to this:
+
+- Speed is nerfed on doubles that are doubletappable. If the time since the last object (deltatime) for the current object is smaller than the 300 hit window, we instead interpolate between that and the deltatime for the previous object.
+- Speed is nerfed if successive objects are faster than the 300 hit window. This is intended to target cheesing of faster bursts and streams if the hit window is way larger than the time between notes.
+
+The intention behind this change is to accommodate for future improvement, as there are a handful of players who have proven themselves to be competent at such high speeds. 
+
+As an added bonus, the star ratings of absurdly fast maps are now measured more accurately, which means you can gawk as much as you’d like at the now even-higher star ratings on them.
+
+### Minor changes
+
+Alongside all those big changes above, there are some changes on the smaller side of things:
+
+- [Sliderbreaks are now approximated into the total miss count](https://github.com/ppy/osu/pull/15086) in performance calculation, following [**StanR**](https://osu.ppy.sh/users/7217455)’s pull request. This is guessed by comparing the achieved combo to the maximum combo possible on a map. 
+- [Instant spinners no longer give an insane amount of strain](https://github.com/ppy/osu/pull/14217), thanks to [**StanR**](https://osu.ppy.sh/users/7217455). This fixes maps [such as this one](https://osu.ppy.sh/beatmapsets/814850#osu/1901200), which abused this. 
+- Joz spotted that the initial strain for skills started at 1, not 0. [This has been fixed](https://github.com/ppy/osu/pull/15014).
+- In preparation for osu!lazer, difficulty calculation for [the Blinds mod](https://github.com/ppy/osu/pull/14921) and [the Relax mod](https://github.com/ppy/osu/pull/14942) has been added by [**Apo11o**](https://osu.ppy.sh/users/9558549).
+  - These could be potentially enabled in the future once osu!lazer leaderboard submissions go live, since the concept of "ranked mods" will be no more.
+
+---
+
+Massive thanks to the star rating developers for all the changes above, the community members for providing continuous feedback, plus [Apo11o](https://osu.ppy.sh/users/9558549) and [Xexxar](https://osu.ppy.sh/users/2773526) for writing this post.
+
+—osu!team
```

Comment:
```suggestion
Massive thanks to the star rating developers for all the changes above, and the community members for providing continuous feedback.

—Apo11o and Xexxar
```

Reply:
These posts are generally always credited to `osu!team` anyhow.

Reply:
Yeah, if anything the previous version read better (leaving `osu!team` as the sign-off). Xexxar is already mentioned above so could probably be left to just Apo11o, or omit completely as this post is going to be a community effort once all these reviews are applied.

Reply:
I commented to point out that it literally says who wrote the post, followed by a sign-off that is different from who wrote the post. I don't have an opinion who counts as the author, just that it shouldn't contradict itself